src/field/launch_application.c

```diff
@@ -0,0 +1,1400 @@
+#include "unk_02092BE8.h"
+#include "unk_02097B78.h"
+#include "save_local_field_data.h"
+#include "save_pokeathlon.h"
+#include "save_special_ribbons.h"
+#include "save_trainer_card.h"
+#include "battle/battle_022378C0.h"
+#include "constants/flags.h"
+
+typedef struct Unk0203E6D4 {
+    HeapID heapId;
+    PartyMenuArgs *partyMenu;
+    u8 unk8[0x4];
+} Unk0203E6D4;
+
+typedef struct Unk0203E878 {
+    Unk0203E8C8 *unk0;
+    u16 *unk4;
+    u16 *unk8;
+    EasyChatArgs *easyChat;
+    int unk10;
+} Unk0203E878;
+
+typedef struct Unk0203F0D0 {
+    Pokemon *mon;
+    SaveFashionDataSub *unk4;
+    FashionCase *fashionCase;
+    Options *options;
+    GAME_STATS *gameStats;
+    PlayerProfile *profile;
+    int *unk18;
+    int unk1C;
+    u32 *unk20;
+} Unk0203F0D0;
+
+typedef struct Unk0203F134 {
+    int state;
+    u16 *unk4;
+    int unk8;
+    Unk0203F0D0 *unkC;
+} Unk0203F134;
+
+typedef struct WirelessTradeSelectMonArgs {
+    FieldSystem *fieldSystem;
+    PlayerProfile *profile;
+    Party *party;
+    SavePalPad *palPad;
+    SaveData *saveData;
+    SaveUnk26 *unk14;
+    Options *options;
+    GAME_STATS *gameStats;
+    Pokedex *pokedex;
+    int unk24;
+    int unk28;
+    BOOL natDexEnabled;
+    int unk30;
+    PlayerProfile *partnerProfile;
+    Pokemon *unk38;
+    Pokemon *unk3C;
+} WirelessTradeSelectMonArgs;
+
+typedef struct TradeSequenceArgs {
+    BoxPokemon *unk0;
+    BoxPokemon *unk4;
+    PlayerProfile *partnerProfile;
+    int bgType;
+    int unk10;
+    Options *options;
+    void *unk18;
+    u8 unk1C[0x4];
+} TradeSequenceArgs;
+
+typedef struct WirelessTradeData {
+    int state;
+    WirelessTradeSelectMonArgs wirelessTradeSelectMon;
+    TradeSequenceArgs tradeSequence;
+} WirelessTradeData;
+
+typedef struct Unk0203F4C8 {
+    struct UnkStruct_0205AC88 *unk0;
+    Options *options;
+} Unk0203F4C8;
+
+typedef struct NamingScreenData {
+    int state;
+    int partyIdx;
+    u16 *retVar;
+    NamingScreenArgs *args;
+    String *unk10;
+} NamingScreenData;
+
+typedef struct Unk0203F844 {
+    int unk0;
+    SysInfo *sysInfo;
+    Party *party;
+    PCStorage *pcStorage;
+    Pokedex *pokedex;
+    UnkStruct_021D2230 *unk14;
+    void *unk18;
+    PlayerProfile *profile;
+    SaveData *saveData;
+    Options *options;
+    GAME_STATS *gameStats;
+    Bag *bag;
+    BOOL natDexEnabled;
+    void *unk34;
+    u32 unk38;
+    int unk3C;
+} Unk0203F844;
+
+typedef struct Unk0203E644 {
+    u8 unk0[0x26];
+    u8 unk26;
+} Unk0203E644;
+
+typedef struct PartyMenuMoveSelectData {
+    HeapID unk0;
+    Unk0203E644 *unk4;
+    PokemonSummaryArgs *pokemonSummary;
+} PartyMenuMoveSelectData;
+
+extern u8 _020FA0B0[];
+
+static PartyMenuArgs *CreatePartyMenuArgs(HeapID heapId, FieldSystem *fieldSystem, int a2, int a3);
+static BOOL Task_OpenPartyMenuThenMoveSelect(TaskManager *taskman);
+static BOOL sub_0203E878(TaskManager *taskman);
+static void _LaunchPokegearPhoneApp(FieldSystem *fieldSystem, PokegearArgs *args);
```

Comment:
these should not have an _

Reply:
I think this is fine -- we use it elsewhere in the codebase (e.g. `_BerryPotsArgs_New`).

Reply:
it shouldn't have a _
underscore prefixes are bad, as for berry pots that should also be changed